src/js/index.js

```diff
@@ -118,6 +118,10 @@ export const hpe = deepFreeze({
         light: '#FFBC44',
       },
       'orange!': '#FF8300',
+      'validation-error': {
```

Comment:
I'm noticing that this color name isn't aligned between the site and Figma. In figma, we show a few validation colors but they're all marked "validation". Let's get resolution from @gregfuruiye or another designer and update the site or Figma where necessary before merging so all of these match

Reply:
Since we are calling most things red 'critical', I wonder if that's what we stick with for anything giving red status. There is another use case where the validation color is being used in notifications for critical alerts, so I'm thinking we just call it validation-critical:

status-critical
validation-critical

Reply:
I think that approach is good! We should just line up figma files with the name